test/test.js

```diff
@@ -395,6 +395,25 @@ describe('serveIndex(root)', function () {
     });
   });
 
+  describe('with "simplifyFileSize" option', function () {
+    it('should simplify file sizes', function (done) {
+      var server = createServer(fixtures, {'simplifyFileSize': true})
+
+      request(server)
+        .get('/collect')
+        .expect(/1\.16 KB/)
+        .expect(200, done)
+    });
+    it('should not simplify file sizes', function (done) {
+      var server = createServer(fixtures, {'simplifyFileSize': false})
+
+      request(server)
+        .get('/collect')
+        .expect(/1186/)
+        .expect(200, done)
+    });
```

Comment:
```suggestion
    it('should not simplify file sizes', function (done) {
      var server = createServer(fixtures, {'simplifyFileSize': false})

      request(server)
        .get('/collect')
        .expect(/1186/)
        .expect(200, done)
    });
    it('should not simplify file sizes by default', function (done) {
      var server = createServer(fixtures)

      request(server)
        .get('/collect')
        .expect(/1186/)
        .expect(200, done)
    });
```

We should include an additional test to specify that this is the default behavior